docs/banner.md

```diff
@@ -1,55 +1,63 @@
-# Alert
+# Banner
 ### Usage
 
 ```jsx
-import { Alert } from 'radiance-ui';
+import { Banner } from 'radiance-ui';
 
-<Alert.Container>
-  <Alert
+<Banner.Container>
+  <Banner
     text="Your photos were uploaded successfully!"
     type="success"
     duration={4}
     onExit={() => {}}
   />
-  <Alert
+  <Banner
     text="Your photos were uploaded successfully!"
     type="success"
     duration="sticky"
     onExit={() => {}}
   />
-  <Alert
+  <Banner
     text="Support has been notified."
     type="info"
     duration="sticky"
     onExit={() => {}}
   />
-  <Alert
+  <Banner
     text="Oops! We couldn't update your information!"
     type="danger"
     duration="sticky"
     onExit={() => {}}
   />
-</Alert.Container>
+</Banner.Container>
+  <Banner
```

Comment:
looks weird to have this out on its own. Wouldn't we want to use this banner inside of the container?